description.md

```diff
@@ -13,6 +13,17 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Media
 
+The `media` content type refers to audio, visual or audiovisual educational materials.
+Images, videos, recordings, educational video games represent examples of such content.
+
+The media content type represents an easier digestible format of presenting information.
+As such, it is typically used in conjunction with other content types, such as reading, slides or demos.
```

Comment:
Elaborate on this that media items are typically embedded within the text destined for reading, practice items, assignments or slides.

Reply:
I did not elaborate on purpose because I was thinking this can be better explained at the "Teaching stages chapter". Of course we can elaborate on both sections

Reply:
Does it have to do with teaching? It's about where the media items fit in, which is still part of the stored content.